test/ntttcp_test.py

```diff
@@ -0,0 +1,441 @@
+        parse_result = ParseNtttcpResult(result.rec_stdout, result.send_stdout)
+        assert parse_result.is_buffer_size(rec_cmd) is True
+
+    def test_bandwidth_limit(self) -> None:
+        common_option = "-B 10G"
+        rec_cmd, send_cmd = self.combine_command(
+                common_option=common_option
+        )
+        result = self.run_test(rec_cmd, send_cmd)
+        parse_result = ParseNtttcpResult(result.rec_stdout, result.send_stdout)
+        bandwidth_limit_pattern = re.compile(r"bandwidth limit \(bits\/sec\):\s+(\d+)")
+        bandwidth_limit = bandwidth_limit_pattern.findall(str(parse_result.send_stdout))
+        cmd_bandwidth_pattern = re.compile(r"-B (\d+)(\w+)")
+        cmd_bandwidth = cmd_bandwidth_pattern.findall(common_option)
+        assert parse_result.is_bandwidth_and_rate_limit(cmd_bandwidth, bandwidth_limit) is True
+
+    def test_output_log_file(self) -> None:
+        common_option = "-O"
+        rec_cmd, send_cmd = self.combine_command(
+                common_option=common_option
+        )
+        result = self.run_test(rec_cmd, send_cmd)
+        parse_result = ParseNtttcpResult(result.rec_stdout, result.send_stdout)
+        assert parse_result.is_output_log_file()
+        throughput = parse_result.get_throughput()
+        print("throughput=", throughput)
+#       #assert throughput >= self.expected_throughput * 0.9
+
+    def test_output_xml_file(self) -> None:
+        common_option = "-x"
+        rec_cmd, send_cmd = self.combine_command(
+                common_option=common_option
+        )
+        result = self.run_test(rec_cmd, send_cmd)
+        parse_result = ParseNtttcpResult(result.rec_stdout, result.send_stdout)
+        assert parse_result.is_output_xml_file()
+        throughput = parse_result.get_throughput()
+        print("throughput=", throughput)
+#       assert throughput >= self.expected_throughput * 0.9
+
+    def test_ouput_json_file(self) -> None:
+        common_option = "-j"
+        rec_cmd, send_cmd = self.combine_command(
+                common_option=common_option
+        )
+        result = self.run_test(rec_cmd, send_cmd)
+        parse_result = ParseNtttcpResult(result.rec_stdout, result.send_stdout)
+        assert parse_result.is_output_json_file()
+        throughput = parse_result.get_throughput()
+        print("throughput=", throughput)
+#        assert throughput >= self.expected_throughput * 0.9
+
+    def test_starting_port_number(self) -> None:
+        common_option = "-p 10000"
+        rec_cmd, send_cmd = self.combine_command(
+                common_option=common_option
+        )
+        result = self.run_test(rec_cmd, send_cmd)
+        parse_result = ParseNtttcpResult(result.rec_stdout, result.send_stdout)
+        assert parse_result.is_starting_port_number(send_cmd) is True
+
+    def test_mapping_option(self) -> None:
+        rec_cmd = "ulimit -n 10240 && ntttcp -D -r -m 200,*,127.0.0.1 -D -t 5 -V"
```

Comment:
If "port" is a judgment, we use a variable.
ports = 200
rec_cmd = f"ulimit -n 10240 && ntttcp -D -r -m {ports},*,127.0.0.1 -D -t 5 -V"
send_cmd = f"ulimit -n 10240 && ntttcp -s127.0.0.1 -P {ports}-t 5"

Reply:
done